README.md

```diff
@@ -17,11 +38,107 @@ Corpus][corpus], as described by Thorsten Brants and Alex Franz, and [distribute
 Linguistic Data Consortium. Note that this data **"may only be used for linguistic
 education and research"**, so for any other usage you should acquire a different data set.
 
-For the microbenchmark included in this repository, instant-segment is ~17x faster than
+For the microbenchmark included in this repository, Instant Segment is ~17x faster than
 the Python implementation. Further optimizations are planned -- see the [issues][issues].
 The API has been carefully constructed so that multiple segmentations can share
 the underlying state to allow parallel usage.
 
+## Installing
+
+### Python **(>= 3.9)**
+
+```sh
+pip install instant-segment
+```
+
+### Rust
+
+```toml
+[dependencies]
+instant-segment = "*"
```

Comment:
This isn't really installing it... Maybe a different section title?

Also, `*` versions selectors aren't commonly used in Rust. I think I'd prefer using the current version number explicitly, even if keeping it up to date is a PITA (not doing so also doesn't detract much from the experience, I think).